### PR TITLE
Add SPIRV projects for blackbox fuzzing with GraphicsFuzz

### PR DESCRIPTION
**Pull request by jonathanmetzman**
cc @afd @paulthomson

### CODE REVIEW

File: projects/graphicsfuzz-spirv/build.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+cmake -G "Ninja" -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_C_COMPILER=$CC -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_CXX_FLAGS="$CXXFLAGS" ..
+ninja
+cp StandAlone/glslangValidator $OUT
+popd
+
+mkdir SPIRV-Tools/build
+pushd SPIRV-Tools/build
+
+# TODO: If cmake respects LDFLAGS, do we need to specify the compilers and their flags?
+# Link failure without LDFLAGS="-lpthread"
```
Comment (afd) [inline projects/graphicsfuzz-spirv/build.sh]: Do the flags to turn on asan already exist in $CFLAGS and $CXXFLAGS?  I guess yes, due to the request for those sanitizers in project.yaml.

Comment (jonathanmetzman) [reply projects/graphicsfuzz-spirv/build.sh]: Yep!

**Comment by oliverchang** [comment]
FYI you need to set up the jobs and blackbox fuzzers manually here. Right now we've allocated bots which are exceptioning out because they haven't been set up.

**Comment by jonathanmetzman** [comment]
Done.

**Pull request merged**